app/pb_kits/playbook/pb_user/user.rb

```diff
@@ -40,6 +41,14 @@ def avatar_size
       def title_size
         size == "lg" ? 3 : 4
       end
+
+      def details
+        if territory.present?
+          [territory, title].compact.join(" • ")
+        else
+          title
+        end
+      end
```

Comment:
You don’t need the constructional. It just needs to be the compact join line.

```ruby
def details
  [territory, title].compact.join(" • ")
end
```

Reply:
Good point, just make sure that the `territory` will not be `""` otherwise you can get to a weird scenario:
```
[1] pry(main)> territory = nil
=> nil
[2] pry(main)> title = "developer"
=> "developer"
[3] pry(main)> [territory, title].compact.join("  - ")
=> "developer"
[4] pry(main)> territory = ""
=> ""
[5] pry(main)> [territory, title].compact.join("  - ")
=> "  - developer"
```

@terryfinn what do you mean by "constructional"?

Reply:
Auto correct mistake, not "constructional".

Meant to say, you don't need the conditional (`if territory.present?`).

Looks like from you example the value for territory and title are not nil, so it would need to be updated to:

```ruby
def details
  [territory, title]reject(&:blank?).join(" • ")
end
```

Reply:
Got it! Now it makes sense!